### PR TITLE
chore: Update links and navigation in Docusaurus

### PR DESCRIPTION
**Pull request by geido**
### SUMMARY
This PR:
- Fixes up the footer links
- Adds copyright for Apache Software Foundation (copy what we have now)
- Adds a link to Slack and Mailing List in the footer
- Removes "Gallery" and "Resources" tabs
- Fixes up the "Edit this page" links to our own repo

### TESTING INSTRUCTIONS
1. Run Docusaurus and verify the above fixes

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: docs-v2/docusaurus.config.js [hunk 1]
```diff
@@ -119,7 +119,7 @@ const config = {
         docs: {
           sidebarPath: require.resolve('./sidebars.js'),
           // Please change this to your repo.
```
Comment (ad-m) [inline docs-v2/docusaurus.config.js:121]: Can we remove that comment as no longer relevant?

**Comment by ad-m** [comment]
I am not aware of any better place to discuss Docusaurus migration implementation. Please indicate such a place (if any), and I will transfer my comments. I have a few comments. 

We have `docs-v2/package-lock.json` and `docs-v2/yarn.lock`. In addiction the project we have different package manager:

```
$ find . -name 'package-lock.json' -o -name 'yarn.lock' | grep -v 'node_modules'
./docs-v2/package-lock.json
./docs-v2/yarn.lock
./superset-websocket/package-lock.json
./superset-websocket/utils/client-ws-app/package-lock.json
./superset-frontend/temporary_superset_ui/superset-ui-plugins-deckgl/yarn.lock
./superset-frontend/temporary_superset_ui/superset-ui/package-lock.json
./superset-frontend/package-lock.json
./superset-frontend/cypress-base/package-lock.json
./docs/package-lock.json
```

* [ ] Can we use only one package manager for Docusaurus project?

* [ ] In long term, can we make consistent which package manager we will gonna use in Superset's project?

In an attempt to build I receive a warning:

```
Invalid docusaurus-plugin-client-redirects version 2.0.0-beta.14.
All official @docusaurus/* packages should have the exact same version as @docusaurus/core (2.0.0-beta.9).
Maybe you want to check, or regenerate your yarn.lock or package-lock.json file?
```

* [ ] Can we sync package versions as @docusaurus/* recommends?

Apache Foundation has "[Website Navigation Links Policy](https://www.apache.org/foundation/marks/pmcs#navigation)". I have the impression that we are not yet aligned with it.

* [ ] Can we add links to "Donate", "Thanks", "Security".

Let me know what you think, I am open to working on the above with you.

**Comment by geido** [comment]
> I am not aware of any better place to discuss Docusaurus migration implementation. Please indicate such a place (if any), and I will transfer my comments. I have a few comments.
> 
> We have `docs-v2/package-lock.json` and `docs-v2/yarn.lock`. In addiction the project we have different package manager:
> 
> ```
> $ find . -name 'package-lock.json' -o -name 'yarn.lock' | grep -v 'node_modules'
> ./docs-v2/package-lock.json
> ./docs-v2/yarn.lock
> ./superset-websocket/package-lock.json
> ./superset-websocket/utils/client-ws-app/package-lock.json
> ./superset-frontend/temporary_superset_ui/superset-ui-plugins-deckgl/yarn.lock
> ./superset-frontend/temporary_superset_ui/superset-ui/package-lock.json
> ./superset-frontend/package-lock.json
> ./superset-frontend/cypress-base/package-lock.json
> ./docs/package-lock.json
> ```
> 
> * [ ]  Can we use only one package manager for Docusaurus project?
> * [ ]  In long term, can we make consistent which package manager we will gonna use in Superset's project?
> 
> In an attempt to build I receive a warning:
> 
> ```
> Invalid docusaurus-plugin-client-redirects version 2.0.0-beta.14.
> All official @docusaurus/* packages should have the exact same version as @docusaurus/core (2.0.0-beta.9).
> Maybe you want to check, or regenerate your yarn.lock or package-lock.json file?
> ```
> 
> * [ ]  Can we sync package versions as @docusaurus/* recommends?
> 
> Apache Foundation has "[Website Navigation Links Policy](https://www.apache.org/foundation/marks/pmcs#navigation)". I have the impression that we are not yet aligned with it.
> 
> * [ ]  Can we add links to "Donate", "Thanks", "Security".
> 
> Let me know what you think, I am open to working on the above with you.

Hey @ad-m thanks for reporting these. I had the same thoughts about the package and yarn locks, as well as that warning. If you think you can solve those problems that'd be great. I can definitely help you with that. Let's chat on Slack sometime soon.

**Comment by geido** [comment]
@ad-m 

> Can we add links to "Donate", "Thanks", "Security".

These I just added. Thanks for noticing and reporting!

**Review by srinify — APPROVED** [review]

**Comment by srinify** [comment]
Hey @ad-m FYI! We ended up merging this PR so we can get Docusaurus out the door (it's been a few months in the making!). The package lock vs yarn lock issue doesn't seem to be blocking so we should collaborate to fix post launch (hopefully early next week).

**Pull request merged**